Create legend_collectors_encounter.nut

Chilly-Maher opened the pull request:
Added an encounter for the collectors situation, story allows for a way to recruit an old soldier but no code has been implemented yet

Chilly-Maher committed:
Update legend_greenskins_situation_encounter.nut

No escape char in dialogue

BreakyZ committed:
Merge pull request #58 from Aloo-Wizard/patch-6

Update legend_greenskins_situation_encounter.nut

Chilly-Maher committed:
Create legend_collectors_encounter.nut

Added an encounter for the collectors situation, story allows for a way to recruit an old soldier but no code has been implemented yet

Chilly-Maher committed:
Rename legend_collectors_encounter.nut to legend_collectors_situation_encounter.nut

renamed to have situation in title

BreakyZ committed:
Update README.md

chopeks committed:
Merge branch 'release/19.1.0' into development

# Conflicts:
#	scripts/encounters/situation/legend_greenskins_situation_encounter.nut

WNTRJimmy committed:
This function was probably copy pasted twice at some point.

chopeks committed:
Merge branch 'release/19.1.0' into development

BreakyZ committed:
Merge pull request #75 from Battle-Brothers-Legends/Function-appears-twice

This function was probably copy pasted twice at some point.

BreakyZ committed:
wip

BreakyZ committed:
Add new military warscythes

BreakyZ committed:
add military warscythes to some entities

BreakyZ committed:
Refactor thrust master and change the effect to work on any melee piercing skill

BreakyZ committed:
Add new run them through perk

hamzah-hayat committed:
Adds extra logging to the OP Scaling Begger

When you gain stats, it tells you in the tactical log.
Fixes Issue https://github.com/Battle-Brothers-Legends/Legends-Bugs/issues/168 😆

chopeks committed:
Merge branch 'release/19.1.0' into development

BreakyZ committed:
Refactor push the advantage and executioner

chopeks committed:
Merge pull request #86 from hamzah-hayat/Scaling-begger-extra-logging

Adds extra logging to the OP Scaling Begger

chopeks committed:
added a chance for legendary contracts to spawn without perk

chopeks committed:
Merge branch 'release/19.1.0' into development

# Conflicts:
#	mod_legends/hooks/skills/actives/prong_skill.nut

chopeks committed:
Merge branch 'development' into feature/legendary_beast_contracts_without_perk

TheCutestSkinghoul committed:
Adds 1 new Tavern name into list of potential names

TheCutestSkinghoul commented on scripts/encounters/situation/legend_collectors_encounter.nut:
```diff
@@ -0,0 +1,33 @@
+    function createScreens() {
+        this.m.Screens.extend([{
+            ID = "Task",
+            Title = "Strolling around the city...",//might make a roll with a small chance to recruit this guy later
```
Good idea

TheCutestSkinghoul commented on scripts/encounters/situation/legend_collectors_encounter.nut:
```diff
@@ -0,0 +1,33 @@
+        this.m.Screens.extend([{
+            ID = "Task",
+            Title = "Strolling around the city...",//might make a roll with a small chance to recruit this guy later
+            Text = "[img]gfx/ui/events/event_17.png[/img]{Near the gates of %settlement%, you see a grizzled old soldier dressed in rags sitting on the stoop of a dilapidated house, head in his hands and weeping quite openly. %SPEECH_ON%He took them! The ungrateful bastard took every last one!%SPEECH_OFF% You would normally be content to walk on past him but the man, with swiftness belying his ag ...
```
**A few things here;**
1) Implication of presence - you don't need to state if the reader saw something when you write it in first person.
> you see a grizzled old soldier -> a grizzled old soldier
> You would normally be content -> Normally, you'd be content...
etc, allow the reader to build the picture in their mind, less is more - avoid being caught in detail for points of a story that do not carry significant weight.

2) Recommendation: ['m' dash or 'em' dash usage](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use) to function as a form of punctuation in longform phrases of sentences.
> After a brief moment, you shrug, and motion for the man to speak. The oldtimers have the most fantastical tales. -> After a brief moment, you shrug, and motion for the man to speak – oldtimers have the most fantastical tales.

3) Avoid the use of phonetics and staggered speech where possible. It can be difficult for a reader to parse.
> [...] has stolen all my possessions, he- he intends to sell it [...] -> [...] has stolen all my possessions... he intends to sell it [...]
> Additionally, this character uses quite formal speech here, but later uses contractions later. I recommend picking and sticking to one as an individual typically has a single mode of speech.

Overall, good event, but the character needs some depth and there's a few nitpicks of mine regarding flow and punctuation. Ping me on Discord if you need more detail or help.

TheCutestSkinghoul reviewed (changes requested):
Please see attached comments inline, I think this is 90% there, just needs some polishing. Good work.

chopeks committed:
Merge branch 'release/19.1.0' into development

chopeks committed:
moved druid encounter to camp

chopeks committed:
renamed legend_recruitment_druid_encounter to legend_recruitment_druid_camp_encounter

Chilly-Maher committed:
Update legend_herbalist_background.nut

typo

chopeks committed:
Merge branch 'release/19.1.0' into development

chopeks committed:
moved preload dir to asset zip

TheCutestSkinghoul committed:
[Savefile safe] Implements a tooltip for the user action of shift + left click hiding an armour layer, which was previously undocumented on UI.

BreakyZ committed:
Add backswing

chopeks committed:
Adds 1 new Tavern name into list of potential names

chopeks committed:
Implements a tooltip for the user action of shift + left click hiding an armour layer, which was previously undocumented on UI.

chopeks committed:
legendary beast contracts without perk

chopeks committed:
rewrote pickX functions to use functional apis

chopeks committed:
changed outfit strings to constants

chopeks committed:
converted ancient helms

BreakyZ committed:
Show beard on the 2 unhold helmets

BreakyZ committed:
Add new specialist perks and some icons

BreakyZ committed:
Fix assassin backgrounds traits and belly dancer now gets an expanded perk tree

chopeks committed:
Merge branch 'release/19.1.0' into development

# Conflicts:
#	scripts/items/shields/legend_tower_shield.nut

chopeks committed:
fixed gobbo xbow to fit convention

chopeks committed:
fixed duplicated arena trait

chopeks committed:
disabled not existing perk

chopeks committed:
fixed borked syntax in coup de grace

chopeks committed:
fixed messed up skill syntax

chopeks committed:
fixed leg

chopeks committed:
integrated compat mod

chopeks committed:
added all helmet outfits def constants

chopeks committed:
replaced helmet string defs with constants

chopeks committed:
added armor defs

chopeks committed:
replaced armor string defs with constants

chopeks committed:
fixed defs i forgot

chopeks committed:
fixed some i missed

chopeks committed:
reworked legend enemies with gear in tactical/enemies

chopeks committed:
reworked legend enemies with gear in tactical/human

chopeks committed:
moved outfit related stuff to outfit config

chopeks committed:
Merge branch 'feature/outfit_rework' into development

# Conflicts:
#	scripts/entity/tactical/humans/legend_free_company_leader.nut

chopeks committed:
Merge branch 'release/19.1.0' into development

chopeks committed:
extracted miniboss logic to a function

chopeks committed:
fixed miniboss logic

BreakyZ committed:
Fix the various bow actives

chopeks committed:
removed not needed file

chopeks committed:
added party size limit logic, fixed troop template serialization

chopeks committed:
flatten config a bit

chopeks committed:
added first step for new scaling

chopeks committed:
forgot to restore original limit

Chilly-Maher committed:
Update belly_dancer_background.nut

Since dancer is hardset to female, edited to match that. Dunno if get gender is defined for this and didn't feel like causing another crisis, so it's all hardcoded. 

I don't think being a mercenary company makes you very familiar with marital advice so I switched that to martial

Chilly-Maher committed:
Update belly_dancer_background.nut

missed a spot

TheCutestSkinghoul committed:
[Savefile soft incompat.] Reworked Lone wolf's hiring rules to allow recruitment of specific backgrounds in towns based on company renown, in divisions. As your renown grows, the more recruits will be present in towns. Higher renown increases quality of hires, but you will start with being unable to hire anyone.

TheCutestSkinghoul committed:
Made corrections to code, removed some backgrounds.

chopeks committed:
Merge pull request #109 from Battle-Brothers-Legends/Luft/lone-wolf-recruitment-rework

Luft/lone wolf recruitment rework

Abysscrane committed:
Update compat_defs a few updates

Abysscrane committed:
Update compat_defs sorry few more updates

chopeks committed:
Merge pull request #108 from Chilly-Maher/patch-38 Update belly_dancer_background.nut description

chopeks committed:
legend_herbalist_background typo

Chilly-Maher committed:
Merge branch 'Battle-Brothers-Legends:development' into patch-9
